gym/envs/classic_control/pendulum.py

```diff
@@ -26,7 +26,7 @@ def seed(self, seed=None):
         self.np_random, seed = seeding.np_random(seed)
         return [seed]
 
-    def step(self,u):
+    def step(self, u, **kwargs):
```

Comment:
I am not a fan of changing `step` call signature - basically, any algorithm that works does not know about this change will not be able to use the new version. 
IMO, a better way of doing this will be the following:
change the constructor to take g:
```python
 def __init__(self, g=10.0):
    self.g = g
    ...
```
This way, you can  create a Pendulum with g=9.8 via:
```python
gym.make('Pendulum-v0', g=9.8)
```
(and make sure in the code below at line 32 `g=10.` is replaced with `g=self.g`)